packages/button/__tests__/Button.test.tsx

```diff
@@ -83,22 +82,19 @@ describe('with props.size', () => {
 })
 
 describe('with props.variant', () => {
-  it.each<Required<ButtonProps<'button'>>['variant']>([
-    'primary',
-    'secondary',
-    'ghost',
-    'overlay',
-    'critical',
-  ])('renders button with variant=%j', (variant) => {
-    render(<Button variant={variant}>{variant}</Button>)
-    expect(screen.getByText(variant)).toBeInTheDocument()
-  })
+  it.each<Required<ButtonProps<'button'>>['variant']>(variants)(
+    'renders button with variant=%j',
```

Comment:
Does the `%j` gets replaced by the value of `variant`?

Reply:
Yes: https://jestjs.io/docs/api#testeachtablename-fn-timeout